pkg/cmd/list_test.go

```diff
@@ -192,16 +342,16 @@ func TestWhoCan_policyRuleMatches(t *testing.T) {
 	}{
 		{
 			scenario: "A",
-			verb:     "get", resource: "service", resourceName: "",
+			verb:     "get", resource: "services", resourceName: "",
```

Comment:
Would it still work for the resource to be singular "service"?

Reply:
Yes. `policyRuleMatcher` does the matching against already resolved `APIResource` which is always plural (e.g. `pods` or `pods/log`). Here I simply made test fixture reflect the implementation. The resolution itself is done by the `ResourceResolver` which supports plural, singular and short names. You can see that in the corresponding `TestResourceResolver_Resolve` test.

So to sum up the following works fine:

```
$ kubectl who-can get pods --subresource=log
$ kubectl who-can get pod --subresource=log
$ kubectl who-can get po --subresource=log
```